api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java

```diff
@@ -62,21 +62,27 @@ public CalculationResultMap evaluate(
       while (iterator.hasNext()) {
         Obs obs = iterator.next();
         if (obs.getVoided()
+            || obs.getValueDatetime() == null
```

Comment:
getValueDate and getValueDateTime return the same value, so use either one, but lets just use one, no need to use both.